### PR TITLE
Disable OwnerRow on TD and RA map props

### LINKED ISSUE #11173: Blossom Trees in TD showing player ownership (release)
http://i.imgur.com/ONAXcHE.png

Spectating is not the issue, is visible while playing.

### PR DESCRIPTION
**Pull request by reaperrr**
...and fix Owners of some trees and tib trees on Lessons from Kosovo and Treasure Island.

Closes #11173.

### CODE REVIEW

File: mods/cnc/rules/defaults.yaml [hunk 1]
```diff
@@ -611,6 +611,7 @@
 	Inherits@1: ^SpriteActor
 	Tooltip:
 		Name: Tree
+		ShowOwnerRow: false
```
Comment (pchote) [inline mods/cnc/rules/defaults.yaml]: Maybe `^Bridge` and `^CivField` too for consistency?

**Comment by reaperrr** [comment]
Updated, disabled OwnerRow on RA props as well.

**Comment by pchote** [comment]
:+1:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/c703bf311464a1666401ae92d302472a626d8fdc)

**Pull request merged**